api/src/routers/review.js

```diff
@@ -0,0 +1,117 @@
+import express from "express";
+import knex from "../database_client.js";
+import { Timestamp } from "mongodb";
```

Comment:
This one seems not to be used?
If not, you can remove it :)

Reply:
Ok Noted